<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Andale Mono</family>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="homeTab">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QLabel" name="homeTabName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>761</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="timerTab">
     <attribute name="title">
      <string>Timer</string>
     </attribute>
     <widget class="QPushButton" name="timerTabStartButton">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>230</y>
        <width>241</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="timerTabStopButton">
      <property name="geometry">
       <rect>
        <x>265</x>
        <y>230</y>
        <width>245</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="timerTabResetButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>230</y>
        <width>241</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="timerTabTimeField">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>27</y>
        <width>763</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>112</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="plainText">
       <string>00:00:00</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="overheadTab">
     <attribute name="title">
      <string>Overhead Pictures</string>
     </attribute>
     <widget class="QGraphicsView" name="overheadTabPictureView">
      <property name="geometry">
       <rect>
        <x>252</x>
        <y>5</y>
        <width>520</width>
        <height>388</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="overheadTabInfoTextField">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>252</x>
        <y>398</y>
        <width>520</width>
        <height>30</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="plainText">
       <string>255.255.255.255</string>
      </property>
     </widget>
     <widget class="QPushButton" name="overheadTabCaptureNewButton">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>242</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Capture New</string>
      </property>
     </widget>
     <widget class="QPushButton" name="overheadTabLighterButton">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>342</y>
        <width>242</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Manual Exposure</string>
      </property>
     </widget>
     <widget class="QPushButton" name="overheadTabDeleteLastButton">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>171</y>
        <width>242</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delete Last</string>
      </property>
     </widget>
     <widget class="QPushButton" name="overheadTabPrintLastButton">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>91</y>
        <width>242</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Print Last</string>
      </property>
     </widget>
     <widget class="QPushButton" name="overheadTabDarkerButton">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>256</y>
        <width>242</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Auto Exposure</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="camerasTab">
     <attribute name="title">
      <string>Cameras</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="videoTab">
     <attribute name="title">
      <string>Video</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="audioTab">
     <attribute name="title">
      <string>Audio</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>751</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Good Morning Wex!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="statusTab">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Status</string>
     </attribute>
     <widget class="QLabel" name="ipAddressLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
       </font>
      </property>
      <property name="text">
       <string>IP Address:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="ipAddressField">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>7</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="plainText">
       <string>255.255.255.255</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="hardwareAddressField">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="plainText">
       <string>FF:FF:FF:FF:FF:FF</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="nameField">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>90</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="plainText">
       <string>01234567890123456</string>
      </property>
     </widget>
     <widget class="QLabel" name="nameLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
       </font>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="hardwareAddressLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
       </font>
      </property>
      <property name="text">
       <string>Hardware Address:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="configServerLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
       </font>
      </property>
      <property name="text">
       <string>Timer Controller:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="timerControllerField">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>130</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="plainText">
       <string>255.255.255.255</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="configTextField">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>351</width>
        <height>411</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="restartButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>196</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Restart</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="statusTabCameraControllerField">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="field6Field">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="field7Field">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="field8Field">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="field9Field">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>330</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="field5Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
       </font>
      </property>
      <property name="text">
       <string>Camera Controller:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="field6Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
       </font>
      </property>
      <property name="text">
       <string>Stuff:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="field7Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
       </font>
      </property>
      <property name="text">
       <string>Stuff:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="field8Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
       </font>
      </property>
      <property name="text">
       <string>Stuff:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="field9Label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>191</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Andale Mono</family>
       </font>
      </property>
      <property name="text">
       <string>Stuff:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshConfigButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>370</y>
        <width>196</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh Config</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
